src/src/WordPosition.java

```diff
@@ -0,0 +1,12 @@
+public class WordPosition {
+    public static void main(String[] args) {
+        System.out.println(findWordPosition("Ararat","rarat"));
+    }
+    public static int findWordPosition(String source, String target) {
+        if (source.contains(target)) {
```

Comment:
Please remove if-else statement. indexOf() method does all the work for you and it can return -1 if the target string is not found.